pkg/tenantctl/controller.go

```diff
@@ -90,8 +107,9 @@ func (c *Controller) getNamespaceManager(config clientConfig) k8s.NamespaceManag
 func (c *Controller) Run(threadiness int, stopCh <-chan struct{}) error {
 	defer utilruntime.HandleCrash()
 	defer c.workqueue.ShutDown()
+	c.namespaceFactory.Start(stopCh)
```

Comment:
To be consistent, please puth the informerfactor `Start` with https://github.com/SAP/stewardci-core/blob/master/cmd/run_controller/main.go#L78